app/views/foreman/job_templates/flatpak_install.erb

```diff
@@ -13,11 +13,19 @@ template_inputs:
   description: Name of the application to install
   input_type: user
   required: true
+- name: Launch a session bus instance
+  description: Launch a session bus instance for the flatpak application using 'dbus-run-session'. Requires package dbus-daemon on client.
+  input_type: user
+  required: false
+  options: "true\r\nfalse"
+  advanced: false
+  value_type: plain
+  default: 'false'
```

Comment:
Should this default be true? It sounds like installing via dbus-session-run works on all systems, but just plain flatpak only works on systems with a graphical interface.

Reply:
That's right but also we need to have dbus-daemon package installed on a client that we run dbus-session-run on.._If_ the non-dbus session install fails, admins will need to install dbus-daemon and re-run the job..

Reply:
Good point, and it looks like not all systems come with dbus-daemon by default.